programs/cli_wallet/main.cpp

```diff
@@ -116,28 +116,27 @@ int main( int argc, char** argv )
          return 0;
       }
 
+      // set up logging
       fc::path data_dir;
       fc::logging_config cfg;
       fc::path log_dir = data_dir / "logs";
-
       fc::file_appender::config ac;
-      ac.filename             = log_dir / "rpc" / "rpc.log";
+      ac.filename             = log_dir / "cli_rpc" / "rpc.log";
       ac.flush                = true;
       ac.rotate               = true;
       ac.rotation_interval    = fc::hours( 1 );
       ac.rotation_limit       = fc::days( 1 );
```

Comment:
I think we have 2 options here for styling the block.

We can do something like:

```
      // set up logging
      fc::path data_dir;
      fc::logging_config cfg;
      fc::path log_dir = data_dir / "logs";
      fc::file_appender::config ac;

      ac.filename             = log_dir / "cli_rpc" / "rpc.log";
      ac.flush                = true;
      ac.rotate               = true;
      ac.rotation_interval    = fc::hours( 1 );
      ac.rotation_limit       = fc::days( 1 );

      cfg.appenders.push_back(fc::appender_config( "default", "console", fc::variant(fc::console_appender::config(), 20)));
      cfg.appenders.push_back(fc::appender_config( "rpc", "file", fc::variant(ac, 5)));

      cfg.loggers                         = { fc::logger_config("default"), fc::logger_config( "rpc") };
      cfg.loggers.front().level           = fc::log_level::info;
      cfg.loggers.front().appenders       = {"default"};
      cfg.loggers.back().level            = fc::log_level::debug;
      cfg.loggers.back().appenders        = {"rpc"};

      std::cout << "Logging RPC to file: " << (data_dir / ac.filename).preferred_string() << "\n";
      fc::configure_logging( cfg );
```

Or just:

```
      // set up logging
      fc::path data_dir;
      fc::logging_config cfg;
      fc::path log_dir = data_dir / "logs";
      fc::file_appender::config ac;
      ac.filename = log_dir / "cli_rpc" / "rpc.log";
      ac.flush = true;
      ac.rotate = true;
      ac.rotation_interval = fc::hours( 1 );
      ac.rotation_limit = fc::days( 1 );
      cfg.appenders.push_back(fc::appender_config( "default", "console", fc::variant(fc::console_appender::config(), 20)));
      cfg.appenders.push_back(fc::appender_config( "rpc", "file", fc::variant(ac, 5)));
      cfg.loggers = { fc::logger_config("default"), fc::logger_config( "rpc") };
      cfg.loggers.front().level = fc::log_level::info;
      cfg.loggers.front().appenders = {"default"};
      cfg.loggers.back().level = fc::log_level::debug;
      cfg.loggers.back().appenders = {"rpc"};
      std::cout << "Logging RPC to file: " << (data_dir / ac.filename).preferred_string() << "\n";
      fc::configure_logging( cfg );
```

Reply:
I broke it up a little bit, separating console logger from file logger. I am trying to find a happy medium between keeping chunks together and readability. Perhaps throw it in a method?